### PR TITLE
Fix DA.Logic.isBool comment

### PR DESCRIPTION
**Pull request by johan-da**

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Thanks!

File: compiler/damlc/daml-stdlib-src/DA/Logic.daml [hunk 1]
```diff
@@ -178,8 +178,8 @@ reduce (Disjunction ds) =
   in foldl work false xs
 
 -- | `isBool` attempts to convert a formula to a bool. It satisfies
--- `isBool true == Right True` and `toBool false == Right False`.
--- Otherwise, it returns `Left x`, where `x` is the input.
+-- `isBool true == Some True` and `toBool false == Some False`.
```
Comment (garyverhaegen-da) [inline compiler/damlc/daml-stdlib-src/DA/Logic.daml:181]:
```suggestion
-- `isBool true == Some True` and `isBool false == Some False`.
```

Comment (johan-da) [reply compiler/damlc/daml-stdlib-src/DA/Logic.daml]: good catch

**Review by remyhaemmerle-da — APPROVED** [review]
Thanks

**Pull request merged**